framework/components/ecl-forms/ecl-forms-labels/ecl-forms-labels.config.js

```diff
@@ -1,5 +1,5 @@
 module.exports = {
   title: 'Labels',
   label: 'Labels',
-  status: 'planned',
+  status: 'ready',
```

Comment:
Please update also the `package.json` as `ecl-forms-help-blocks` for example to be publishable.